src/controllers/__tests__/profileController.test.ts

```diff
@@ -56,14 +56,14 @@ const proxyUserResponse = {
   spid_email: anEmailAddress,
   token: "123hexToken",
   version: 1 as NonNegativeInteger,
-  last_app_version: "0.0.1"
 };
 
 const apiUserProfileResponse = {
   email: anEmailAddress,
   is_email_validated: true,
   is_inbox_enabled: true,
   is_webhook_enabled: true,
+  reminder_status: "UNSET",
```

Comment:
Should not happens this case. `fn-app` never returns `UNSET` and valid `ReminderStatus` value are only `ENABLED` and `DISABLED`

Reply:
you're right I removed the default in https://github.com/pagopa/io-backend/pull/930/commits/cf4518b25adc6c9cbfeb14c25d7420d8db587346